### PR TITLE
fix(Path): [Android] fix certain paths being skewed

### PR DESCRIPTION
**Pull request by kazo0**
closes https://github.com/unoplatform/nventive-private/issues/191

## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the current behavior?
MicrosoftTeams-image (1)

## What is the new behavior?
MicrosoftTeams-image

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

**Comment by ajpinedam** [comment]
Wondering if there is a way to Test this.

**Review by ajpinedam — APPROVED** [review]

File: src/Uno.UI/UI/Xaml/Shapes/Shape.Android.cs [hunk 1]
```diff
@@ -63,15 +63,12 @@ protected override void OnDraw(Canvas canvas)
 				return;
 			}
 
-			var translateMatrix = new Android.Graphics.Matrix();
-			var scaleMatrix = new Android.Graphics.Matrix();
+			var matrix = new Android.Graphics.Matrix();
```
Comment (davidjohnoliver) [inline src/Uno.UI/UI/Xaml/Shapes/Shape.Android.cs:66]: Is this related to the fix or just a refactoring?

**Review by davidjohnoliver — APPROVED** [review]

File: src/Uno.UI/UI/Xaml/Shapes/Shape.Android.cs (hunk 1 above)
Comment (kazo0) [reply src/Uno.UI/UI/Xaml/Shapes/Shape.Android.cs:66]: It was an initial attempt at a fix but turns out to just be a refactoring.
